src/test/java/uk/gov/companieshouse/company_appointments/CompanyAppointmentServiceTest.java

```diff
@@ -419,6 +434,137 @@ void testPatchCompanyNameStatusThrowsUnsupportedOperationException() {
         assertThrows(UnsupportedOperationException.class, executable);
     }
 
+    @Test
+    @DisplayName("Should update appointment with no exceptions thrown")
+    void testSuccessfulUpdate() throws ServiceUnavailableException {
+        // given
+        when(companyStatusValidator.isValidCompanyStatus(anyString())).thenReturn(true);
+        when(fullRecordAppointmentRepository.patchAppointmentNameStatus(anyString(), anyString(), anyString(), any(), anyString())).thenReturn(1L);
+
+        // when
+        Executable executable = () -> companyAppointmentService.patchNewAppointmentCompanyNameStatus(COMPANY_NUMBER, APPOINTMENT_ID, COMPANY_NAME, OPEN_STATUS, CONTEXT_ID);
+
+        // then
+        assertDoesNotThrow(executable);
+        verify(companyStatusValidator).isValidCompanyStatus(OPEN_STATUS);
+        verify(resourceChangedApiService).invokeChsKafkaApi(new ResourceChangedRequest(CONTEXT_ID, COMPANY_NUMBER, APPOINTMENT_ID, null, false));
+        verify(fullRecordAppointmentRepository).patchAppointmentNameStatus(any(), any(), any(), any(), any());
+    }
+
+    @Test
```

Comment:
Can a display name be added to this test please?

Reply:
Addressed in latest commit